### PR TITLE
Fix BDW DMA timer

### PR DESCRIPTION
**Pull request by xiulipan**
Fix BDW DMA timer.
Fix typo in BDW SSP issue.

Thanks @tlauda to help debug the timer issue.
Fix https://github.com/thesofproject/sof/issues/2096

### CODE REVIEW

File: src/platform/haswell/platform.c [hunk 1]
```diff
@@ -202,9 +202,9 @@ int platform_init(struct sof *sof)
 	trace_point(TRACE_BOOT_PLATFORM_CPU_FREQ);
```
Comment (kv2019i) [inline src/platform/haswell/platform.c]: @xiulipan Patch looks fine, but I don't think it's correct to name this as a "typo". The same error is twice here, both in comment and in code, so does not seem like a typo, but simply a bug.

Comment (xiulipan) [reply src/platform/haswell/platform.c]: OK, I will make it into a fix. Good point.

**Comment by jajanusz** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
CI usual suspects.

**Pull request merged**